### ISSUE TITLE
Model fitting accuracy

### ISSUE THREAD

**Issue by Zhijing1128** [hmmlearn/hmmlearn#212]
Hi,

I run the basic code and found the fitting accuracy is low.

Here's my code:

np.random.seed(42)

model = hmm.GaussianHMM(n_components=3, covariance_type="full")
model.startprob_ = np.array([0.6, 0.3, 0.1])
model.transmat_ = np.array([[0.7, 0.2, 0.1], [0.3, 0.5, 0.2], [0.3, 0.3, 0.4]])
model.means_ = np.array([[0.0, 0.0], [3.0, -3.0], [5.0, 10.0]])
model.covars_ = np.tile(np.identity(2), (3, 1, 1))
X, Z = model.sample(100000)
print(X)

remodel = hmm.GaussianHMM(n_components=3, covariance_type="full", n_iter=30000)
remodel.fit(X)
print(remodel.startprob_)

The remodel.startprob_ is very different from model.startprob. Why? Are the iteration # and trained # enough? 

thanks.

**Comment by superbobry** [comment]
`startprob_` is estimated only from the first observations of the sequence. So if you learn on a single sequence (as you indeed do in your example), you only use a single observation for estimating `startprob_`. In order to estimate it better you need to fit the model to multiple sequences.
